packages/appstore/src/components/wallet-account/wallet-account.tsx

```diff
@@ -0,0 +1,160 @@
+};
+
+const WalletAccount = ({ account }: WalletAccountProps) => {
+    const isAccountVirtual = account.is_virtual === 1;
+
+    return (
+        <Accordion
+            expand_section={false}
+            elevation_type='xs'
+            onToggle={() => {
+                /**do something */
+            }}
+            type='containerWithBorder'
+            className='wallet-account'
+        >
+            <Icon className='wallet-account__background' icon='IcAppstoreDemoWallet' />
+            <Accordion.Title size='medium'>
+                <div className='wallet-account__header'>
+                    <div className='wallet-account__details'>
+                        <div className='wallet-account__logo'>
+                            <WalletCard wallet_name={isAccountVirtual ? 'demo' : ''} size='small' />
+                        </div>
+                        <div className='wallet-account__info'>
+                            <Text bold type='subtitle-2'>
+                                <Localize
+                                    i18n_default_text='Demo {{currency}} wallet'
+                                    values={{ currency: account.currency }}
+                                />
+                            </Text>
+                            <div className='wallet-account__actions'>
+                                <WalletActionButton size='small' label='Transfer' icon='icAppstoreTransfer' />
+                                <WalletActionButton size='small' label='Transaction' icon='icAppstoreTransaction' />
+                                <WalletActionButton size='small' label='Reset Balance' icon='icAppstoreResetBalance' />
+                            </div>
+                        </div>
+                    </div>
+                    <div className='wallet-account__balance'>
+                        {isAccountVirtual && (
+                            <Text bold={false} type='small' align='right'>
+                                {localize('Virtual balance')}
+                            </Text>
+                        )}
+                        <Text bold type='subtitle-1'>
+                            <Money amount={account.wallet?.balance} currency={account.currency} show_currency />
+                        </Text>
+                    </div>
+                </div>
+            </Accordion.Title>
+            <Accordion.Content size='medium'>
+                <div className='wallet-account__content'>
+                    <div>
+                        <Text bold type='subtitle-1'>
+                            {localize('CFDs')}
+                        </Text>
+                        <Text bold={false} type='paragraph-1'>
+                            {localize(
+                                'Trade with leverage and tight spreads for better returns on successful trades. '
+                            )}
+                            <a href='#' className='link-text'>
```

Comment:
Can we use the `<Text>` component here since the `link-text` class is only adding a color to the text?

Reply:
@carol-binary what about the link? this text should be a link and the default link style doesn't match the design

Reply:
Ahh I see your point, can be wrapped in <Text> but no worries, you can consider this a nit comment.